testimony/__init__.py

```diff
@@ -124,6 +124,9 @@ def _parse_docstring(self):
                 else:
                     self.skipped_lines.append(line)
 
```

Comment:
I think is better to improve the docstring and say that if `@test` is not available the first line will be get instead.